src/adera/Machines/Rocket.cpp

```diff
@@ -61,16 +61,15 @@ WireOutput* MachineRocket::request_output(WireOutPort port)
 
 std::vector<WireInput*> MachineRocket::existing_inputs()
 {
-    /*std::vector<WireInput*> inputs;
+    std::vector<WireInput*> inputs;
```

Comment:
seems a bit weird to use a raw pointer for something that you know is always non-null.

std::reference_wrapper might be closer to what you really want.

Reply:
@Capital-Asterisk what do you think of this? I was just following the convention of the other `existing_inputs()` functions, should we change all of these to references instead of pointers?

Reply:
I want the entire wiring system dead, so there isn't really a point. Changing the syntax won't untangle the spaghetti.

Reply:
Yeah fair enough